tests/testthat.R

```diff
@@ -1,4 +1,5 @@
 library(testthat)
 library(archive)
+library(tools)
```

Comment:
Please do not load tools here, but qualify references with `::` in the test files.

Reply:
Fixed; I used R for 15 years but I didn't even know the feature.